packages/figma-design-tokens/input/tokens/$themes.json

```diff
@@ -2344,4 +2348,4 @@
     },
     "group": "Sizes"
   }
-]
+]
```

Comment:
Can this 'no newline at end of file' can be resolved. Please check in other files as well.

Reply:
this comes from token studio. we will always have this occur when tokens are changed on design side. our formatter isn't configured to run on token input files.